test/docs/validation.test.js

```diff
@@ -384,6 +384,53 @@ describe('validation docs', function() {
     // acquit:ignore:end
   });
 
+  it('Cast Error Message Overwrite', function() {
+    const vehicleSchema = new mongoose.Schema({
+      numWheels: {
+        type: Number,
+        cast: '{VALUE} is not a number'
+      }
+    });
+    const Vehicle = db.model('Vehicle', vehicleSchema);
+
+    const doc = new Vehicle({ numWheels: 'pie' });
+    const err = doc.validateSync();
+
+    err.errors['numWheels'].name; // 'CastError'
+    // "pie" is not a number
+    err.errors['numWheels'].message;
+    // acquit:ignore:start
+    assert.equal(err.errors['numWheels'].name, 'CastError');
+    assert.equal(err.errors['numWheels'].message,
+      '"pie" is not a number');
+    db.deleteModel(/Vehicle/);
+    // acquit:ignore:end
+  });
+
+  /* eslint-disable no-unused-vars */
```

Comment:
after the next version of `acquit-ignore` would be released, it would also be possible to just disable for a specific line
```ts
// acquit:ignore-next-line
// eslint-disable-next-line no-unused-vars
const doc = new Vehicle({ numWheels: 'pie' });
```

[`ignore-next-line`](https://github.com/vkarpov15/acquit-ignore#also-supports-a-ignore-next-line)

Reply:
I'll patch that up once we release the next version of acquit-ignore